packages/framework-provider-local-infrastructure/src/scheduler.ts

```diff
@@ -8,9 +8,14 @@ interface ScheduledCommandInfo {
 
 export function configureScheduler(config: BoosterConfig, userProject: any): void {
   const triggerScheduleCommand = userProject['boosterTriggerScheduledCommand']
+
+  const isEmpty = (obj: {}) => {
+    return Object.keys(obj).length === 0
+  }
+
   Object.keys(config.scheduledCommandHandlers)
     .map((scheduledCommandName) => buildScheduledCommandInfo(config, scheduledCommandName))
-    .filter((scheduledCommandInfo) => scheduledCommandInfo.metadata.scheduledOn)
+    .filter((scheduledCommandInfo) => scheduledCommandInfo.metadata.scheduledOn && !isEmpty(scheduledCommandInfo.metadata.scheduledOn))
```

Comment:
I believe the existing condition in this predicate filters out elements where `scheduledCommandInfo.metadata.scheduledOn` is "[false-ish](https://developer.mozilla.org/en-US/docs/Glossary/Truthy)", to avoid calling `scheduler.scheduleJob` for cron expressions that are undefined, null, etc:

This 2nd condition will filter out `scheduledOn` that are empty (`{}`), to avoid calling the scheduler on the resulting undefined expression.